com.unity.formats.alembic/Runtime/Scripts/Exporter/AlembicExporter_impl.cs

```diff
@@ -275,10 +273,42 @@ public void SetupSubmeshes(aeObject abc, Mesh mesh)
                     return;
                 }
 
-                points.LockList(ls => mesh.GetVertices(ls));
+                if (world2local != Matrix4x4.identity)
+                {
+                    var verts = new List<Vector3>();
+                    mesh.GetVertices(verts);
+                    for (var i = 0; i < verts.Count; ++i)
+                    {
+                        var v = verts[i];
+                        verts[i] = world2local.MultiplyPoint(v);
+                    }
+
+                    points.Assign(verts);
```

Comment:
Can't you call LockList instead of Assign. It seems Assign is making a memcpy whereas LockList is avoiding the extra copy